selenium_tests/course_info/page_objects/course_people_page_object.py

```diff
@@ -68,15 +59,17 @@ def search_and_add_user(self, user_id, role):
         # Enter user to search on
         self.find_element(*Locators.ADD_PEOPLE_SEARCH_TXT).send_keys(user_id)
         # Select role
-        self.select_role_type(role)
+        self.select_role_type(canvas_role)
 
         # Click 'Add to course' course button
         self.find_element(*Locators.ADD_TO_COURSE_BUTTON).click()
+        WebDriverWait(self._driver, 30).until_not(lambda s: s.find_element(
```

Comment:
Yeah, we ran into the issue where tests were failing because users have not yet been added.  This primarily adds additional "buffer" time (assuming when progress bar disappears, then it's likely user has been added).  If it times out, the next assert should still catch it and throw an error as before -- "assert self.assertTrue(self.people_page.is_person_on_page(test_user))".